packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -336,25 +336,27 @@ export class ChatContext {
    * @returns Result of calling `f`. Also re-raises any exceptions thrown from `f`.
    * @throws ChatError. Exceptions thrown from `f` are tagged with the failed `target.
    */
-  public async asyncTeeErrorToState<T>(
-    f: () => Promise<T>,
+  public withAsycnErrorTeedToState<Args extends unknown[], R>(
```

Comment:
nit typo asycn -> async

Reply:
Heh. A good IDE lets me make the same typo, every time :)

Reply:
Done.